tests/integration/models/reddit/test_subreddit.py

```diff
@@ -1473,6 +1474,99 @@ def test_submissions__with_pause_and_skip_after(self, _):
             assert submission_count < 100
 
 
+class TestSubredditModerationStreams(IntegrationTest):
+    def setup_reddit(self):
+        self.reddit = Reddit(
+            client_id=pytest.placeholders.client_id,
+            client_secret=pytest.placeholders.client_secret,
+            refresh_token=pytest.placeholders.refresh_token,
+            redirect_uri=pytest.placeholders.redirect_uri,
+            user_agent=pytest.placeholders.user_agent,
+        )
```

Comment:
Now that we have #1233, this can be removed.

Reply:
oops missed that